lib/setup.py

```diff
@@ -1,16 +1,22 @@
+import os
 import platform
 import setuptools
 import subprocess
+import sys
 
 from pipenv.project import Project
 from pipenv.utils import convert_deps_to_pip
-from sys import version_info
+from setuptools.command.install import install
+
+VERSION = "0.56.0"  # PEP-440
+
+NAME = "streamlit"
 
 pipfile = Project(chdir=False).parsed_pipfile
 
 # Combine [packages] with either [python3] or [python2]
 packages = pipfile["packages"].copy()
-if version_info.major == 2:
+if sys.version_info.major == 2:
```

Comment:
Is this still needed?

Reply:
I don't think it's needed but I'll leave it for now since @nthmost is working on removing python 2 stuff in #1177

Reply:
Yeah, that'll get pulled out today.